File: HA3_3.java
//Write a program which accept number from user and display all its non factors
//Input : 12
//Output: 5    7    8   9   10  11
import java.util.*;

class Number
{
    public void NonFact(int iNo)
    {
        int iCnt = 0;

        for(iCnt = 1; iCnt <= iNo; iCnt++)
        {
            if((iNo % iCnt)!=0)
            {
                System.out.println(iCnt);
            }
        }
    }
}

public class HA3_3 
{
    public static void main(String[] args) 
    {
        Scanner sobj = new Scanner(System.in);
        
        System.out.println("Enter a number");
        int iValue = sobj.nextInt();

        Number nobj = new Number();
        {
            nobj.NonFact(iValue);
        }
        sobj.close();
    }
}

File: HA3_3.py
#Write a program which accept number from user and display all its non factors
#Input : 12
#Output: 5    7    8   9   10  11

def NonFact(iNo):
    for iCnt in range (1, iNo, 1):
        if((iNo % iCnt)!=0):
            print(iCnt)

def main():
    iValue = 0

    print("Enter a number")
    iValue = int(input())

    NonFact(iValue)

if __name__ == "__main__":
    main()